cloudtrail-check.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+# 
+# This script is here to automate (as much as possible) the tasks in
+# https://cloud.gov/docs/ops/maintenance-list/#review-aws-cloudtrail-events
+# 
+# authorized events will include:
+#   a user name like i-17deadbeef1234567
+#   a source IP address within AWS.
+#   an AWS access key starting with ASIA
+# 
+# XXX The source IP range is kind of janky.  It would be better if this
+#     pulled dynamically from AWS and also if we handled cidr notation
+#     properly.  But these two ranges seem to work for govcloud right now.
+
+START=$(date -v -15d +"%Y-%m-%dT%H:%M:%SZ")
+echo "pulling cloudtrail from a bit over 2 weeks ago ($START)"
```

Comment:
Let's be precise about the date range here.

Reply:
I am.  I expect that there will be some fuzz factor for when the operator runs this, so if we just do 2 weeks, we risk not examining data if one operator ran it at 9am on monday of their first shift, and the next operator runs it at 10am on monday of their first shift.  I figured that 1 day of fuzz factor would be sufficient.

Reply:
I think what @jmcarp is getting at is, if it's 15 days, say 15 days, not "a bit over 2 weeks ago"